dev/io.openliberty.microprofile.reactive.messaging.internal_fat/fat/src/io/openliberty/microprofile/reactive/messaging/fat/kafka/validation/KafkaValidationTests.java

```diff
@@ -0,0 +1,148 @@
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.reactive.messaging.fat.kafka.validation;
+
+import static com.ibm.ws.microprofile.reactive.messaging.fat.kafka.framework.KafkaTestClientProvider.CONNECTION_PROPERTIES_KEY;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.PropertiesAsset;
+import com.ibm.ws.microprofile.reactive.messaging.fat.kafka.common.ConnectorProperties;
+import com.ibm.ws.microprofile.reactive.messaging.fat.kafka.framework.KafkaTestClientProvider;
+
+import componenttest.annotation.AllowedFFDC;
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import io.openliberty.microprofile.reactive.messaging.fat.suite.KafkaTests;
+import io.openliberty.microprofile.reactive.messaging.fat.suite.ReactiveMessagingActions;
+import io.openliberty.microprofile.reactive.messaging.fat.validation.AppValidator;
+
+@RunWith(FATRunner.class)
+@AllowedFFDC
+public class KafkaValidationTests {
+
+    private static final String SERVER_NAME = "SimpleRxMessagingServer";
+
+    @Server(SERVER_NAME)
+    public static LibertyServer server;
+
+    @ClassRule
+    public static RepeatTests r = ReactiveMessagingActions.repeat(SERVER_NAME,
+                                                                  ReactiveMessagingActions.MP61_RM30,
+                                                                  ReactiveMessagingActions.MP50_RM30,
+                                                                  ReactiveMessagingActions.MP60_RM30);
+
+    @BeforeClass
+    public static void setup() throws Exception {
+        server.startServer();
+    }
+
+    @AfterClass
+    public static void shutdown() throws Exception {
+        server.stopServer("CWMRX1006E");
+    }
+
+    @Test
+    public void testMissingConnector() throws Exception {
+        // Configure delivery to a non-existant connector
+        ConnectorProperties incoming = ConnectorProperties.simpleIncomingChannel(KafkaTests.connectionProperties(), "missingConnector", ValidConfig.INCOMING_CHANNEL, "Connector",
+                                                                                 "Topic");
+        PropertiesAsset appConfig = new PropertiesAsset().include(incoming);
+
+        AppValidator.validateAppOn(server)
+                        .withClass(ValidConfig.class)
+                        .withAppConfig(appConfig)
+                        .failsWith("Unknown connector for `" + ValidConfig.INCOMING_CHANNEL + "`.")
```

Comment:
should try with a regex similar to `"SRMSG00072*" + ValidConfig.INCOMING_CHANNEL` that uses the Weld  Msg Code and the incoming channel in case the rest of the message content does change for some reason in the future